bigtable/docs/snippets_table.py

```diff
@@ -387,5 +396,88 @@ def test_bigtable_table_row():
     table.truncate(timeout=300)
 
 
+def test_bigtable_batcher_mutate_flush_mutate_rows():
+    # [START bigtable_batcher_mutate]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+    # Batcher for max row bytes, max_row_bytes=1024 is optional.
+    batcher = table.mutations_batcher(max_row_bytes=1024)
+
+    # Add a single row
+    row_key = b'row_key_1'
+    row = table.row(row_key)
+    row.set_cell(COLUMN_FAMILY_ID,
+                 COL_NAME1,
+                 'value-0',
+                 timestamp=datetime.datetime.utcnow())
+
+    # In batcher, mutate will flush current batch if it
+    # reaches the max_row_bytes
+    batcher.mutate(row)
+    batcher.flush()
+    # [END bigtable_batcher_mutate]
+
+    # [START bigtable_batcher_flush]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+    # Batcher for max row bytes, max_row_bytes=1024 is optional.
+    batcher = table.mutations_batcher(max_row_bytes=1024)
+
+    # Add a single row
+    row_key = b'row_key'
+    row = table.row(row_key)
+    row.set_cell(COLUMN_FAMILY_ID, COL_NAME1, 'value-0')
+
+    # In batcher, mutate will flush current batch if it
+    # reaches the max_row_bytes
+    batcher.mutate(row)
+    batcher.flush()
+    # [END bigtable_batcher_flush]
+
+    rows_on_table = []
+    for row in table.read_rows():
+        rows_on_table.append(row.row_key)
+    assert len(rows_on_table) == 2
+    table.truncate(timeout=200)
+
+    # [START bigtable_batcher_mutate_rows]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+    batcher = table.mutations_batcher()
+
+    row1 = table.row(b'row_key_1')
+    row2 = table.row(b'row_key_2')
+    row3 = table.row(b'row_key_3')
+    row4 = table.row(b'row_key_4')
+
+    row1.set_cell(COLUMN_FAMILY_ID, COL_NAME1, b'cell-val1')
+    row2.set_cell(COLUMN_FAMILY_ID, COL_NAME1, b'cell-val2')
+    row3.set_cell(COLUMN_FAMILY_ID, COL_NAME1, b'cell-val3')
+    row4.set_cell(COLUMN_FAMILY_ID, COL_NAME1, b'cell-val4')
+
+    batcher.mutate_rows([row1, row2, row3, row4])
+
+    # batcher will flush current batch if it
```

Comment:
It would be better if this comment read a bit more like a sentence as will display it.

Perhaps something like "`batcher.mutate_rows` will flush the current batch if it	reaches the max flush_count"?